packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -190,18 +193,15 @@ export const PaymentForm = ({
       </FieldGroup>
 
       {confirm && plan && (
-        <Checkbox
-          data-testid="confirm"
-          name="confirm"
-          required
-          label={`
-          I authorize Mozilla, maker of Firefox products, to charge my
-          payment method $${formatCurrencyInCents(plan.amount)} per ${
-            plan.interval
-          }, according to payment
-          terms, until I cancel my subscription.
-        `}
-        />
+        <Checkbox data-testid="confirm" name="confirm" required>
+          I authorize Mozilla, maker of Firefox products, to charge my payment
+          method{' '}
+          <strong>
+            ${`${formatCurrencyInCents(plan.amount)}`} per
+            {` ${plan.interval}`}
```

Comment:
`${formatCurrencyInCents(plan.amount)} per {plan.interval}` should give the same output.

If we're not in a rush to get this merged it might be nice to have a test for this to make sure it properly renders `children`. You could create a mock test in `fields.test.tsx` and essentially duplicate this test but with the new way that you're using it.
```
describe('Checkbox', () => {
  it('renders its own label', () => {
```

Could call it something like `renders its own label with children`. I can jump in here and do this for you too if you're busy the next couple of days.